src/components/SaveKeys.vue

```diff
@@ -2,69 +2,89 @@
 .full-width
   .row.justify-center.q-gutter-lg
     .col-auto
-      q-icon(color="blue-3" name="vpn_key" size="80px")
+      q-icon(color="blue-3", name="vpn_key", size="80px")
```

Comment:
These comas are super weird in this context, this is not how HTML (from which pug syntax is derived) works

Reply:
I figured out these are called `attribute separators`. The default was **comma** in prettier (vetur is using prettier for formatting pug fields). So a new file is introduced called ".prettierrc", along with these commas, all attribute separators are deleted. Is it fine now?